packages/d3fc-axis/src/axisOrdinal.js

```diff
@@ -37,10 +40,117 @@ const axisOrdinal = (orient, scale) => {
         return { offset: [0, base.tickPadding()] };
     };
 
+    const decorateRotation = s => {
+        if (labelRotate) {
+            const { rotate, labelHeight, anchor } = rotation;
+
+            const text = s.select('text');
+            const existingTransform = text.attr('transform');

```

Comment:
just spotted this - grabbing an existing attribute from the DOM, and modifying / supplementing it feels like risky business. When re-applying this below, I guess this is via the D3 update selection? should it also be applied to enter selections?